### PR TITLE
Added related exhibition content fields

### PR DESCRIPTION
**Pull request by gestchild**
references #3009

### CODE REVIEW

File: prismic-model/js/exhibitions.js [hunk 1]
```diff
@@ -32,8 +32,34 @@ const Exhibitions = {
...
+  Resources: {
+    resources: list('Resources', {
+      resource: link('Resource', 'document', ['exhibition-resources'])
+    })
+  },
+  Related: {
+    relatedOfs: list('Related of', {
+      relatedOf: link('In this exhibition', 'document', ['installations', 'events'])
+    }),
+    relatedAbouts: list('Related about', {
```
Comment (jamesgorrie) [inline prismic-model/js/exhibitions.js]: As per discussions (and this is just me reiterating, so if we think it's good, we're good) - but talking to @silveroliver, we're not saying it's related. We know the explicit relationship, so should expose it as such.

Installations for example don't go in here, exhibitions are exhibits within an exhibition, which could also have works exhibited in it for example.

File: prismic-model/js/exhibitions.js [hunk 2]
```diff
@@ -32,8 +32,34 @@ const Exhibitions = {
...
   },
-  // When we get a better relational model up, this will be deprecated
-  'Related promos': {
+  Resources: {
+    resources: list('Resources', {
+      resource: link('Resource', 'document', ['exhibition-resources'])
+    })
+  },
+  Related: {
+    relatedOfs: list('Related of', {
```
Comment (jamesgorrie) [inline prismic-model/js/exhibitions.js]: The language of "related" is also asking content editors to put exactly that in, where as what we're modelling is what we have, not whether someone thinks things are similar.

File: common/services/prismic/exhibitions.js [hunk 3]
```diff
@@ -248,14 +249,15 @@ function putPermanentAfterCurrentExhibitions(exhibitions: UiExhibition[]): UiExh
...
-      exhibitionResourcesFields
-    )
+    fetchLinks: [
+      ...peopleFields,
+      ...contributorsFields,
+      ...placesFields,
+      ...installationFields,
+      ...exhibitionResourcesFields,
+      ...booksFields
+    ]
```
Comment (jamesgorrie) [inline common/services/prismic/exhibitions.js]: 👍

File: common/services/prismic/exhibitions.js [hunk 4]
```diff
@@ -262,6 +271,42 @@ export async function getExhibition(req: Request, id: string): Promise<?UiExhibi
...
+    peopleFields,
+    contributorsFields,
+    eventSeriesFields,
+    eventPoliciesFields,
+    contributorsFields
+  );
+  const extraContent = await getTypeByIds(null, types, ids, {fetchLinks});
+  const parsedContent = extraContent.results.map(doc => {
+    switch (doc.type) {
+      case 'books' :
```
Comment (jamesgorrie) [inline common/services/prismic/exhibitions.js]: We shouldn't be doing books?

File: common/services/prismic/exhibitions.js [hunk 5]
```diff
@@ -262,6 +271,42 @@ export async function getExhibition(req: Request, id: string): Promise<?UiExhibi
   }
 }
 
+// TODO better naming
```
Comment (jamesgorrie) [inline common/services/prismic/exhibitions.js]: Is there something this is doing, that this doesn't?
https://github.com/wellcometrust/wellcomecollection.org/blob/master/common/services/prismic/multi-content.js

File: common/services/prismic/exhibitions.js [hunk 6]
```diff
@@ -161,10 +172,7 @@ function parseExhibitionDoc(document: PrismicDocument): UiExhibition {
     textAndCaptionsDocument: textAndCaptionsDocument,
     featuredImageList: promos,
     resources: Array.isArray(data.resources) ? parseResourceTypeList(data.resources, 'resource') : [],
-    relatedBooks: promoList.filter(x => x.type === 'book').map(parsePromoListItem),
```
Comment (davidpmccormick) [inline common/services/prismic/exhibitions.js]: decision to revert this so old exhibitions will still render their dumb promos.

File: common/services/prismic/parsers.js [hunk 7]
```diff
@@ -307,30 +307,6 @@ export function parseNumber(fragment: PrismicFragment): number {
   return parseInt(fragment, 10);
 }
 
-type PrismicPromoListFragment = {|
```
Comment (davidpmccormick) [inline common/services/prismic/parsers.js]: decision to revert this so old exhibitions will still render their dumb promos.

**Pull request merged**